cmd/summarizer/main.go

```diff
@@ -84,10 +84,11 @@ func main() {
 		defer cancel()
 		return summarizer.Update(ctx, client, opt.config, opt.concurrency, opt.dashboard, opt.confirm)
 	}
-
+	start := time.Now()
 	if err := updateOnce(ctx); err != nil {
 		logrus.WithError(err).Error("Failed update")
 	}
+	logrus.Infof("updateOnce took %d ms to complete\n", int64(time.Since(start)))
```

Comment:
time.Duration is the correct type here, not int64. Either way, ms is the wrong units for something which takes many minutes and sometimes more than an hour.

Look at https://golang.org/pkg/time/#Duration.Round

Also you can already calculate this information by looking at the deltas between the Sleeping log lines on L102.

Reply:
Oh okay, thanks for the heads up. I guess since we already have this functionality then I'll just remove this. But I'll definitely keep that in mind for the future!